fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/PopulateResourceTypes.java

```diff
@@ -66,26 +82,34 @@ public void run(IDatabaseTranslator translator, Connection c) {
                 Properties props = new Properties();
                 props.load(fis);
 
+                int numToProcess = 0;
                 for (Entry<Object, Object> valueEntry : props.entrySet()) {
                     Integer curVal = Integer.parseInt((String) valueEntry.getValue());
                     String resource = (String) valueEntry.getKey();
-                    batch.setInt(1, curVal);
-                    batch.setString(2, resource);
-                    batch.addBatch();
-                }
 
-                // Check Error Codes.
-                int[] codes = batch.executeBatch();
-                int errorCodes = 0;
-                for (int code : codes) {
-                    if (code < 0) {
-                        errorCodes++;
+                    if (values.containsKey(resource)) {
```

Comment:
Shouldn't this be 
```
   if (!values.containsKey(code))
```

Reply:
ack... how did this work locally... I'll fix this up

Reply:
Eagle Eyes again! thank you